src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/client/ProfessionalApiClient.java

```diff
@@ -778,4 +778,18 @@ public Object retrieveAllActiveOrganisationsWithMinimalInfo(RequestSpecification
             return response.getBody().as(ErrorResponse.class);
         }
     }
+
+    public Map<String, Object> findMFAByUserId(HttpStatus expectedStatus, String userId) {

```

Comment:
As we are getting the value from our DB instead of calling any external API, I think it is not necessary to have a functional test case. I believe integration test case would suffice. @abhijitdiwan please correct me if I am wrong here

Reply:
@shreedhar-hmcts - you can correct me too if I am wrong here

Reply:
its is better to bare minimum 1 functional test case always for any api because it interacts with sidam ,s2s etc and tested before production deployment. Yes all other negative scenarios can be covered in integration test case.

Reply:
Thanks @shreedhar-hmcts for the comment. 
@ishratt  - As suggested by @shreedhar-hmcts , you can leave it as such and no need to remove. Thanks